osu.Game.Rulesets.Mania/Timing/Drawables/DrawableScrollingTimingChange.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Rulesets.Timing;
+
+namespace osu.Game.Rulesets.Mania.Timing.Drawables
+{
+    public class DrawableScrollingTimingChange : DrawableManiaTimingChange

```

Comment:
also one-line xmldoc wouldn't go astray (just describing what the "scroll" part of this is actually achieving)